src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -9319,3 +9321,86 @@ TEST(convolution_f32_gpu, convolution_gpu_bfyx_f16_depthwise_x_bloxk_size_1) {
 TEST(export_import_convolution_f32_gpu, convolution_gpu_bfyx_f16_depthwise_x_bloxk_size_1) {
     test_convolution_f32_gpu_convolution_gpu_bfyx_f16_depthwise_x_bloxk_size_1<FLOAT16>(true);
 }
+
+
+TEST(convolution_f32_fw_gpu, basic_convolution_no_bias_swap_xy) {
```

Comment:
if i remember correctly, you said fusion test is required. isn't it?

Reply:
Added.